web/templates/topic_link.hbs

```diff
@@ -1,2 +1,13 @@
-<a class="stream-topic" data-stream-id="{{channel_id}}" href="{{href}}">#{{channel_name}} &gt; {{topic_display_name}}</a>
-{{~!-- squash whitespace --~}}
+{{#if is_empty_string_topic}}
+<a class="stream-topic" data-stream-id="{{channel_id}}" href="{{href}}">
```

Comment:
We can remove the `data-stream-id` here as we no longer use this. But keeping it as a follow-up task to first verify that we really no longer use it anywhere else in the web-client.

Reply:
Yeah, that seems good.